pages/common/git-add.md

```diff
@@ -19,6 +19,10 @@
 
 `git add -f`
 
-- Add parts of a file interactively:
+- Interactively stage parts of changes to your tracked files:
```

Comment:
I would keep the description short and similar to the previous one. Can we change this to - "Interactively stage parts of files", and make the next one "Interactively stage parts of a given file" ?